app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupAutoFillViewModel.kt

```diff
@@ -16,9 +18,13 @@ import javax.inject.Inject
 @HiltViewModel
 class SetupAutoFillViewModel @Inject constructor(
     private val settingsRepository: SettingsRepository,
+    private val authRepository: AuthRepository,
 ) :
     BaseViewModel<SetupAutoFillState, SetupAutoFillEvent, SetupAutoFillAction>(
-        initialState = SetupAutoFillState(dialogState = null, autofillEnabled = false),
+        initialState = run {
+            val userId = requireNotNull(authRepository.userStateFlow.value).activeUserId
+            SetupAutoFillState(userId = userId, dialogState = null, autofillEnabled = false)
```

Comment:
I wonder if the `autofillEnabled` flag should be populated by this: 

```kotlin
settingsRepository.isAutofillEnabledStateFlow.value
```

This state is based on the app being set regardless of number of users, so it could be enabled by a previous user.

Reply:
need confirm but I think the collection in the init block does this if that is the case. If not I will create ticket for that specific issue if thats cool and move this change fwd.

Reply:
yep confirmed!